### PR TITLE
Bau log at appropriate level

### PR DESCRIPTION
**Pull request by oswaldquek**

### CODE REVIEW

File: src/main/java/uk/gov/pay/connector/gateway/GatewayClient.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public GatewayClient(Client client, MetricRegistry metricRegistry) {
                 return gatewayResponse;
             } else {
                 if (statusCode >= INTERNAL_SERVER_ERROR.getStatusCode()) {
-                    LOGGER.error("Gateway returned unexpected status code: {}, for gateway url={} with type {} with order request type {}",
+                    LOGGER.info("Gateway returned unexpected status code: {}, for gateway url={} with type {} with order request type {}",
```
Comment (alexbishop1) [inline src/main/java/uk/gov/pay/connector/gateway/GatewayClient.java]: A gateway returning a 5xx feels like it should be at least a warning.

File: src/main/java/uk/gov/pay/connector/gateway/smartpay/auth/SmartpayAccountSpecificAuthenticator.java [hunk 2]
```diff
@@ -32,7 +34,7 @@ public SmartpayAccountSpecificAuthenticator(GatewayAccountDao gatewayAccountDao,
                 .filter((gatewayAccountEntity) -> matchCredentials(basicCredentials, gatewayAccountEntity))
                 .map(gatewayAccountEntity -> Optional.ofNullable(gatewayAccountEntity.getNotificationCredentials().toBasicAuthUser()))
                 .orElseGet(() -> {
-                    logger.error(format("Authentication failure: failed for smartpay username %s", basicCredentials));
+                    LOGGER.info(format("Authentication failure: failed for smartpay username %s", basicCredentials));
```
Comment (alexbishop1) [inline src/main/java/uk/gov/pay/connector/gateway/smartpay/auth/SmartpayAccountSpecificAuthenticator.java]: I think this is an error. It suggests a service has got their credentials wrong, which we can help them fix.

File: src/main/java/uk/gov/pay/connector/gateway/stripe/handler/StripeCaptureHandler.java [hunk 3]
```diff
@@ -70,23 +70,24 @@ public CaptureResponse capture(CaptureGatewayRequest request) {
...
+                LOGGER.info("Capture failed for transaction id {}. Failure code from Stripe: {}, failure message from " +
+                                "Stripe: {}. External Charge id: {}. Response code from Stripe: {}",
                         transactionId, errorCode, errorMessage, request.getExternalId(), e.getStatus());
 
                 return fromBaseCaptureResponse(new StripeCaptureResponse(transactionId, errorCode, errorMessage), null);
             }
 
             if (e.getFamily() == SERVER_ERROR) {
-                logger.error("Capture failed for transaction id {}. Reason: {}. Status code from Stripe: {}. Charge External Id: {}",
+                LOGGER.info("Capture failed for transaction id {}. Reason: {}. Status code from Stripe: {}. Charge External Id: {}",
```
Comment (alexbishop1) [inline src/main/java/uk/gov/pay/connector/gateway/stripe/handler/StripeCaptureHandler.java]: This is probably OK if we’re going to retry (and eventually log an error if the retries are exhausted).

**Review by alexbishop1 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/pay/connector/gateway/smartpay/auth/SmartpayAccountSpecificAuthenticator.java (hunk 2 above)
Comment (oswaldquek) [reply src/main/java/uk/gov/pay/connector/gateway/smartpay/auth/SmartpayAccountSpecificAuthenticator.java]: Hmm, how about WARN? As I understand it, an ERROR is a serious issue and represents the failure of something important going on in your application.

Comment (alexbishop1) [reply src/main/java/uk/gov/pay/connector/gateway/smartpay/auth/SmartpayAccountSpecificAuthenticator.java]: Yes, I think warn would be appropriate. We can still alert on that.

File: src/main/java/uk/gov/pay/connector/gateway/stripe/handler/StripeCaptureHandler.java (hunk 3 above)
Comment (oswaldquek) [reply src/main/java/uk/gov/pay/connector/gateway/stripe/handler/StripeCaptureHandler.java]: Probably best to WARN actually, I'll change it.

File: src/main/java/uk/gov/pay/connector/gateway/GatewayClient.java (hunk 1 above)
Comment (oswaldquek) [reply src/main/java/uk/gov/pay/connector/gateway/GatewayClient.java]: Fixed

**Review by alexbishop1 — APPROVED** [review]

**Pull request merged**